### PR TITLE
Fix on demand radio Amp player position

### LINKED ISSUE #7073: Amp player - Scrub bar current-position-indicator is cut off
On amp, on OD Radio and Live Radio pages, the scrub bar current position indicator/dogegg is cut off:

Image 30-06-2020 at 11 52

It is fine on canonical

On the galaxy s10+ you can also see the duration cut off
Image 30-06-2020 at 14 08

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #7073

**Overall change:**
Fix on demand radio Amp player position

**Code changes:**
- Remove margin for amp

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by rebeccamcginn** [comment]
It fixes the current time indicator but creates an issue for line up:

image

**Comment by pharingee** [comment]
@rebeccarfmcginnbbc I've updated the issue a bit. It's still a little off, but then if it goes further, duration and scrub-bar will be cut off

**Review by RichardPK — COMMENTED** [review]
Placement still doesn't look quite right? This is with the `indonesia/bbc_indonesian_radio/w13xtt0s.amp` asset on Chrome.

More noticeable on mobile

Screenshot 2020-07-07 at 12 52 08

But also on desktop:

Screenshot 2020-07-07 at 12 52 49

File: src/app/pages/OnDemandRadioPage/index.jsx [hunk 1]
```diff
@@ -63,7 +63,11 @@ const StyledMessageContainer = styled.div`
   margin-bottom: ${GEL_SPACING_QUAD};
 `;
 
+// iframe padding set to keep scrub bar and duration in view
 const StyledAudioPlayer = styled(AVPlayer)`
+  amp-iframe {
+    overflow: visible !important;
```
Comment (rebeccamcginn) [inline src/app/pages/OnDemandRadioPage/index.jsx:69]:
```suggestion
    overflow: visible !important;
    width: calc(100% + ${GEL_SPACING_DBL});
    @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
      width: calc(100% + ${GEL_SPACING_QUAD});
    }
```

Comment (rebeccamcginn) [reply src/app/pages/OnDemandRadioPage/index.jsx:69]: That should resolve the above @RichardPK

Comment (RichardPK) [reply src/app/pages/OnDemandRadioPage/index.jsx:69]: Awesome looks like that did it

**Review by RichardPK — APPROVED** [review]

**Review by rebeccamcginn — APPROVED** [review]

**Pull request merged**